### ISSUE TITLE
When using --dir with --source-maps the "sources" entries are relative to the cwd not the output directory

### ISSUE THREAD

**Issue by nuisanceofcats** [google/traceur-compiler#1777]
```
traceur --dir srcdir outdir
```

After this the map files in "outdir" should be prefixed with "../srcdir/" but currently they are prefix with "srcdir/"

**Comment by nuisanceofcats** [comment]
Same for the "file" entries.

**Comment by johnjbarton** [comment]
> After this the map files in "outdir" should be prefixed with "../srcdir/" 

I don't understand what that means.  The map file is in the `outdir`.

**Comment by nuisanceofcats** [comment]
The ".map" files sit alongside the target files in the output directory yet the "sources" and "files" file paths are relative to the directory the traceur compiler was run from, they should be relative to the output directory.

**Comment by sagivf** [comment]
I agree - If you build in one machine and port the code to another, Or even move it a directory - it breaks the path and the map resource will not load.

**Comment by nuisanceofcats** [comment]
@sagivf The problem is even more fundamental than that. I'd recommend trying `babel`, there are several more significant issues of this nature with `traceur`.
